lib/models/message.ts

```diff
@@ -281,4 +289,45 @@ export class MessageModel extends DocumentDbModel<
       query: "SELECT * FROM messages m WHERE (m.fiscalCode = @fiscalCode)"
     });
   }
+
+  public async attachStoredContent(
+    messageId: string,
+    partitionKey: string,
+    message: IMessageWithContent
+  ): Promise<Either<Error, Option<IRetrievedMessage>>> {
+    const blobId = this.getMessageAttachmentName(messageId);
+
+    // store media (attachment) with message content in blob storage
+    const errorOrStoredMessage = await upsertBlobFromText(
```

Comment:
Using just `Message` may be confusing (in other parts of the code `Message` is just the metadata), better refer to `MessageContent` in this case: `errorOrMessageContent`

Reply:
ok